source/upgrade.txt

```diff
@@ -94,9 +111,7 @@ Version 5.0 Breaking Changes
   In earlier versions, this parameter is of type ``int`` for both methods. To view an example
   that shows how to call ``SocketSettings`` methods, see the :ref:`SocketSettings Example
   <java-socketsettings-example>` in the Specify MongoClient Settings
-  guide.
-
-  This change breaks binary compatibility (requires recompiling) but does not
+  guide. This change breaks binary compatibility (requires recompiling) but does not
   require code changes.
```

Comment:
Suggestion: I think we generally try to avoid putting statements in parenthesis like this if not necessary.

```suggestion
  guide. This change breaks binary compatibility and requires recompiling, but does not
  require code changes.
```